index.js

```diff
@@ -6,12 +6,17 @@ import querystring from 'querystring';
 const fs = require('fs');
 const express = require('express');
 const bodyParser = require('body-parser');
-const swaggerMiddleware = require('swagger-express-middleware');
 const interceptor = require('express-interceptor');
 
+const isOpenApi3 = !!process.env.IS_OAS3;
+const getApiSpec = req => (isOpenApi3 ? req.openapi : req.swagger);
+const swaggerMiddleware = isOpenApi3
+  ? require('swagger-express-middleware-3')
+  : require('swagger-express-middleware-2');
+
 const app = express();
 const port = process.env.PORT;
-const swaggerPath = process.env.SWAGGER_SPEC;
+const swaggerPath = process.env.SWAGGER_SPEC || process.env.API_SPEC; // TODO deprecate SWAGGER_SPEC
```

Comment:
im thinking i'll deprecate SWAGGER_SPEC before merging. making this breaking but this is already too risky to not be a breaking change 🤷‍♂